hydra-node/json-schemas/api.yaml

```diff
@@ -97,6 +111,52 @@ channels:
           - $ref: "#/components/messages/Contest"
           - $ref: "#/components/messages/Fanout"
 
+  /commit:
+    description: Main entry point to draft commit transaction in Hydra service.
+    servers:
+      - localhost-http
+    subscribe:
+      summary: Hydra node drafted commit transaction responses.
+      operationId: draftCommitTxResponse
+      message:
+        $ref: "#/components/messages/DraftCommitTxResponse"
+      bindings:
+        http:
+          type: response
+          method: POST
+          query:
```

Comment:
Must: not use `query`

The response does use a query parameters, but only a response body.